ONSdigital/design-system
# Feature/2603 tab labels as headings

Issue - [#2603](https://github.com/ONSdigital/design-system/issues/2603)
### What is the context of this PR?
Changed the html content inside `example-tabs.html` to render an additional `h2` element when component is rendered at a viewport with less that 740px width. This is to allow better user orientation on smaller devices.

### How to review
1. Spin up design system locally, pull latest changes, checkout to branch.
2. View tabs component example - `example-tabs.html`.
3. Inspect element and confirm tab titles are being displayed as headings <= 740px viewport width.

### Example 
![Screenshot 2023-05-09 at 10 53 56](https://user-images.githubusercontent.com/131372109/237061577-54897779-9d65-4a66-9e42-8ecfc3139efe.png)
